chia/wallet/puzzles/dao_proposal.clsp

```diff
@@ -330,7 +352,7 @@
             YES_VOTES
             TOTAL_VOTES
             PROPOSED_PUZ_HASH
-            vote_coin_ids_or_proposal_timelock_length
+            (distinct_or_raise vote_coin_ids_or_proposal_timelock_length ())
```

Comment:
`loop_over_vote_coins` is already iterating over the `vote_coin_ids` list. If you integrated the `distinct_or_raise` check as part of that function you could save looping over that list twice